l2geth/core/state_transition_ovm.go

```diff
@@ -100,6 +100,11 @@ func EncodeSimulatedMessage(msg Message, timestamp, blockNumber *big.Int, execut
 		to = &common.Address{0}
 	}
 
+	value := msg.Value()
+	if value == nil {
+		value = big.NewInt(0)
```

Comment:
Requesting that @tynes take a look at this section of code.

Reply:
We could save an allocation by doing:

```
	if value == nil {
		value = common.Big0
```

`value` is not mutated anywhere else so its safe to use this global

Reply:
updated to use this